File: CS/WindowsApplication103/ToolTipObject.cs
using System;
using System.Collections.Generic;
using System.Text;
using DevExpress.XtraBars;

namespace WindowsApplication103
{
    class ToolTipObject
    {
        public string ToolTip;
        public BarButtonItem ToolTipItem;
        public string ToolTipCustomizationMode;
        public ToolTipObject(BarButtonItem _toolTipItem, String _toolTip,string _toolTipCustocustomizationMode)
        {
            ToolTipItem = _toolTipItem;
            ToolTip = _toolTip;
            ToolTipCustomizationMode = _toolTipCustocustomizationMode;
        }

    }
}

File: VB/WindowsApplication103/ToolTipObject.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports DevExpress.XtraBars

Namespace WindowsApplication103
	Friend Class ToolTipObject
		Public ToolTip As String
		Public ToolTipItem As BarButtonItem
		Public ToolTipCustomizationMode As String
		Public Sub New(ByVal _toolTipItem As BarButtonItem, ByVal _toolTip As String, ByVal _toolTipCustocustomizationMode As String)
			ToolTipItem = _toolTipItem
			ToolTip = _toolTip
			ToolTipCustomizationMode = _toolTipCustocustomizationMode
		End Sub

	End Class
End Namespace
